packages/ui/src/lib/NativeBase/Navbar/index.stories.tsx

```diff
@@ -0,0 +1,38 @@
+import { ComponentStory, ComponentMeta } from "@storybook/react";
+import React from "react";
+
+import NativeBase from "..";
```

Comment:
Isn't `NativeBase` `null`?

Reply:
No, it will import NativeBase from 'src/lib/NativeBase/index.tsx'.

Reply:
Hey, I just realised that Navbar should be inside lib not NativeBase

Reply:
lib/NativeBase is just an example for developers to reference

Reply:
Sure, I will delete it and copy Providor from flareon.

Reply:
Sorry to have confused you :(

Reply:
XD never mind. We have a consensus now.